src/pages/StockConfiguration/UseCaseConfiguration/Sections.js

```diff
@@ -0,0 +1,82 @@
+import i18n from '@dhis2/d2-i18n'
+import { ButtonStrip, TabBar, Tab } from '@dhis2/ui'
+import findIndex from 'lodash/findIndex'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { Button } from '../../../components'
+import Details from './Details'
+import General from './General'
+import styles from './Sections.module.css'
+import Transactions from './Transactions'
+
+export const Sections = ({ settings, handleSettings, element }) => {
+    const [selectedTab, setSelectTab] = useState(0)
+
+    const TabItems = [
+        {
+            key: 'general-tab',
+            label: i18n.t('General'),
+            content: General,
+        },
+        {
+            key: 'details-tab',
+            label: i18n.t('Details'),
+            content: Details,
+        },
+        {
+            key: 'transactions-tab',
+            label: i18n.t('Transactions'),
+            content: Transactions,
+        },
+    ]
+
+    const transactionKey = findIndex(TabItems, { key: 'transactions-tab' })
+
+    const CurrentSection = TabItems[selectedTab].content
+
+    return (
+        <>
+            <div className={styles.tabs}>
+                <TabBar fixed>
+                    {TabItems.map(({ label }, index) => (
+                        <Tab
+                            key={index}
+                            onClick={() => setSelectTab(index)}
```

Comment:
Navigation by clicking on the header tab does not work and the mouse cursor is disabled when hovering over the header. Can you take a look? Thanks!

Reply:
Yes, the tabs are disabled. The user can only add more settings in this first version of the app. The second and the third tab depend on the program selected in the first tab. There is a "Next" button but not a "Prev" to prevent changing the program.